packages/orchestrator-ui-components/src/components/WfoForms/formFields/types.ts

```diff
@@ -53,3 +53,19 @@ export interface ServicePort {
     nonremovable?: boolean;
     modifiable?: boolean;
 }
+
```

Comment:
Can this be moved to a surf folder? They are very SURF specific.

Reply:
Create a surf folder in the form directoryt and moved them there